.travis.yml

```diff
@@ -44,6 +44,7 @@ script:
   - |
     if [ "$TRAVIS_OS_NAME" = linux ]; then
       export TEST_DEBUG=1
+      docker tag $CONTROLLER_IMAGE ${CONTROLLER_IMAGE_NAME}:latest
```

Comment:
It would be better if we used a tag that doesn't exist in the repo, so we can be sure we don't run into issues if the image pull policy changes. I would use `TRAVIS_BUILD_NUMBER`.

Reply:
I had thought about that but then I ended up with the `latest` because it reflects the situation that we use "latest" version for testing